### ISSUE TITLE
Audio Switcher Client Issue

### ISSUE THREAD

**Issue by xenolightning** [xenolightning/AudioSwitcher_v1#1018]
### Details:
Audio Switcher
Version: 1.6.4.2
Operating System: Microsoft Windows NT 6.2.9200.0 64-bit
Administrator Privileges: False

### User Comments:
Removed phone jack from from front panel.
### Stacktrace:
```
System.NullReferenceException: Referência de objeto não definida para uma instância de um objeto.
   em AudioSwitcher.AudioApi.CoreAudio.CoreAudioDevice.get_IsDefaultDevice()
   em FortyOne.AudioSwitcher.AudioSwitcher.RefreshNotifyIconItems()
   em FortyOne.AudioSwitcher.AudioSwitcher.RefreshPlaybackDevices()
```

**Comment by miguelsantos** [comment]
Wrote it wrong in the user comment. I disconnected the mic from the front pannel.

**Comment by xenolightning** [comment]
This one should be fixed in 1.6.5.1.

Same issue as #930 and #950
